packages/kyt-core/package.json

```diff
@@ -62,23 +63,24 @@
     "shelljs": "0.7.8",
     "source-map-support": "0.4.11",
     "strip-ansi": "3.0.1",
-    "style-loader": "0.18.1",
+    "style-loader": "0.23.1",
     "stylelint": "9.9.0",
     "stylelint-config-kyt": "1.0.0-alpha.2",
     "stylelint-config-standard": "18.2.0",
     "temp": "0.8.3",
     "url-loader": "0.5.8",
-    "webpack": "3.1.0",
-    "webpack-assets-manifest": "0.7.0",
-    "webpack-dev-middleware": "1.11.0",
-    "webpack-dev-server": "2.5.1",
-    "webpack-hot-middleware": "2.18.2",
-    "webpack-merge": "4.1.0",
-    "webpack-node-externals": "1.6.0",
-    "webpack-plugin-hash-output": "^1.3.0"
+    "webpack": "3.12.0",
+    "webpackbar": "3.1.5",
+    "webpack-assets-manifest": "1.0.0",
+    "webpack-dev-middleware": "2.0.6",
+    "webpack-dev-server": "2.11.3",
+    "webpack-hot-middleware": "2.24.3",
+    "webpack-merge": "4.1.5",
+    "webpack-node-externals": "1.7.2",
+    "webpack-plugin-hash-output": "1.3.1"
   },
   "peerDependencies": {
-    "babel-polyfill": "6.26.0"
+    "kyt-runtime": "pre"
```

Comment:
Should we make this "beta" ahead of the merge to master and then publish a beta? Or do you want to do it in a follow up?